app/services/form1010cg/service.rb

```diff
@@ -11,6 +11,21 @@ class InvalidVeteranStatus < StandardError
 
     NOT_FOUND = 'NOT_FOUND'
 
+    def self.metrics
+      prefix = 'api.form1010cg'
```

Comment:
probably a nitpick, but most of the other service classes define a STATSD_KEY_PREFIX constant. STATSD_KEY_PREFIX= 'api.form1010cg'

Reply:
^^ Just to keep this uniform across the codebase

Reply:
Good call! I've updated this.